.babelrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
- [ ] It would be better if we remove some of the files or assets not needed for this project. For example; 
```
logo192.png logo512.png manifest.json robots.txt
```
and its associated link path in the index.html file